frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-section-card-title-header/common-section-card-title-header.component.html

```diff
@@ -23,8 +23,5 @@ <h5 class="title ft-18-700">
         {{ ('feature-flags.details-updated-at.text' | translate) + (updatedAt | formatDate: 'short date') + '.' }}
       </span>
     </ng-container>
-    <ng-container *ngIf="showViewLogs">
-      <a class="view-logs-link ft-14-400" [href]="" (click)="viewLogsClicked()">View logs</a>
-    </ng-container>
```

Comment:
why are you removing it from the common component? this can be used for other use cases as well.

Reply:
Where else can it be used other than showing the "View Logs" link?

Reply:
Not sure, but we can add the view logs link later after adding filters too

Reply:
Can we re-add it later when we need it again? I think leaving it might confuse us, and we might not need it soon.